week-4/C-more-complex-objects/exercise-1.js

```diff
@@ -4,30 +4,53 @@
   Make sure you run the file after and it outputs the correct results.
 */
 
+//this is a schema and variable data in data the customer has inputted refers to this schema and stores that info in the way this is organised.
+
 var house = {
-    address: "1 Kinning Park",
-    previousOwners: ["Claire M.", "John A."],
-    currentOwner: {
-      firstName: "Margaret",
-      lastName: "Conway"
-    }
+  address: "1 Kinning Park",
+  previousOwners: ["Claire M.", "John A."],
+  currentOwner: {
+    firstName: "Margaret",
+    lastName: "Conway"
+  },
+  changeAddress: function(newAddress) {
+    return (this.address = newAddress);

```

Comment:
Great idea to use an setter method 👏🏻 It seems like you decided not to use it though?

Minor improvement: you don't need the parentheses here.